### PR TITLE
Feature/fullscreen

### PR DESCRIPTION
**Pull request by danielbarboza**
## Goal
Allow only fullscreen by the app

## How to test 
1 . Run all automated tests
2. Open sample app and try to put the video on fullscreen

### CODE REVIEW

File: Sources/Clappr/Classes/Base/Options.swift [hunk 1]
```diff
@@ -4,7 +4,6 @@ public let kPosterUrl = "posterUrl"
 public let kSourceUrl = "sourceUrl"
 public let kFullscreen = "fullscreen"
 public let kFullscreenDisabled = "fullscreenDisabled"
```
[the remark this answers is not in the document]
Comment (geisonfgf) [reply Sources/Clappr/Classes/Base/Options.swift:6]: The problem with the suggestion of kDisableFullscreenButton name is that, in fact, it is not the only change in the player when it is true also a back button is displayed. The idea of this option is to disable the fullscreen functionality forced disconsidering other options related.

**Review by VitorFerraz — APPROVED** [review]

File: Sources/Clappr/Classes/Base/Options.swift (hunk 1 above)
[the remark this answers is not in the document]
Comment (BruunoMM) [reply Sources/Clappr/Classes/Base/Options.swift]: This option disables the fullscreen button and **only if** the player is in Fullscreen, then the back button is shown. An app could still enable fullscreen programmatically. I agree with @torives that we could rename it to make it more meaningful and semantically correct.

I think `kFullscreenDisabled` does not represent its true meaning. Maybe `kDisableFullscreenUserInteraction`?

File: Sources/Clappr/Classes/Base/Core.swift [hunk 2]
```diff
@@ -118,7 +114,7 @@ open class Core: UIObject, UIGestureRecognizerDelegate {
     private func bindEventListeners() {
         #if os(iOS)
         listenTo(self, eventName: InternalEvent.userRequestEnterInFullscreen.rawValue) { [weak self] _ in self?.fullscreenHandler?.enterInFullscreen() }
-        listenTo(self, eventName: InternalEvent.userRequestExitFullscreen.rawValue) { [weak self] _ in self?.onUserRequestExitFullscreen() }
+//        listenTo(self, eventName: InternalEvent.userRequestExitFullscreen.rawValue) { [weak self] _ in self?.onUserRequestExitFullscreen() }
```
[the remark this answers is not in the document]
Comment (danielbarboza) [reply Sources/Clappr/Classes/Base/Core.swift]: Already removed.

File: Sources/Clappr/Classes/Base/Options.swift (hunk 1 above)
[the remark this answers is not in the document]
Comment (danielbarboza) [reply Sources/Clappr/Classes/Base/Options.swift]: changed from `kFullscreen` to `kStartInFullscreen` and `kFullscreenDisabled` to `kDisableFullscreenButton`

File: Sources/Clappr/Classes/Base/Options.swift [hunk 3]
```diff
@@ -34,7 +33,6 @@ public let kMetaDataArtwork = "mdArtwork"
 struct OptionsUnboxer {
```
[the remark this answers is not in the document]
Comment (danielbarboza) [reply Sources/Clappr/Classes/Base/Options.swift]: Removed the unnecessary "unboxer"

**Review by BruunoMM — APPROVED** [review]

**Pull request merged**